---
layout: page
title: Research
permalink: /research/
---

Over 170 publications, including refereed workshop and conference publications, can be found on my Google Scholar profile [here](http://scholar.google.com/citations?user=XKCyZk0AAAAJ&sortby=pubdate). 
The symbol ✝ denotes current/former student/postdoc supervised by me, solely or jointly, when the research was done.

<p><strong>Selected Publications</strong></p>

<ol reversed>
  <li>
    <p>N Doudchenko, M Zhang✝, E Drynkin, EM Airoldi, V Mirrokni, J Pouget-Abadie. <a href="https://arxiv.org/abs/2010.02108"><strong>Causal inference with bipartite designs</strong></a>. <strong><em>arXiv</em></strong> no. 2010.02108, 2020.</p>
  </li>
  <li>
    <p>D Tran✝, P Toulis✝, EM Airoldi. <a href="https://cran.r-project.org/web/packages/sgd/vignettes/sgd-jss.pdf"><strong>Stochastic gradient descent methods for parameter estimation at scale</strong></a>. <strong><em>Journal of Statistical Software</em></strong>. In press, 2020.</p>
  </li>
  <li>
    <p>P Toulis✝, T Horel✝, EM Airoldi. <strong>The Proximal Robbins-Monro Method</strong>. <strong><em>Journal of the Royal Statistical Society, Series B</em></strong>. In press, 2020. (IBM Best student paper award at the New England Statistics Symposium 2018.)</p>
  </li>
  <li>
    <p>L Forastiere✝, EM Airoldi, F Mealli. <a href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2020.1768100"><strong>Identification and estimation of treatment and interference effects in observational studies on networks</strong></a>. <strong><em>Journal of the American Statistical Association</em></strong>. In press, 2020.</p>
  </li>
  <li>
    <p>A Ghasemian, H Hosseinmardi, A Galstyan, EM Airoldi, A Clauset. Stacking Models for Nearly Optimal Link Prediction in Complex Networks. Proceedings of the National Academy of Sciences. In press, 2020.</p>
  </li>
</ol>

**Selected Publications**   

<ol reversed>
  <li>
  <p>   N Doudchenko, M Zhang✝, E Drynkin, EM Airoldi, V Mirrokni, J Pouget-Abadie. [**Causal inference with bipartite designs**](https://arxiv.org/abs/2010.02108). ***arXiv*** no. 2010.02108, 2020.
  </p>
  </li>

<li>
  <p>     D Tran✝, P Toulis✝, EM Airoldi. [**Stochastic gradient descent methods for parameter estimation at scale**](https://cran.r-project.org/web/packages/sgd/vignettes/sgd-jss.pdf). ***Journal of Statistical Software***. In press, 2020.
  </p>
  </li>

<li>
  <p>     P Toulis✝, T Horel✝, EM Airoldi. **The Proximal Robbins-Monro Method**. ***Journal of the Royal Statistical Society, Series B***. In press, 2020. (IBM Best student paper award at the New England Statistics Symposium 2018.)
  </p>
  </li>

<li>
  <p>    L Forastiere✝, EM Airoldi, F Mealli. [**Identification and estimation of treatment and interference effects in observational studies on networks**](https://www.tandfonline.com/doi/abs/10.1080/01621459.2020.1768100). ***Journal of the American Statistical Association***. In press, 2020.</p>
  </li>

<li>
  <p>    A Ghasemian, H Hosseinmardi, A Galstyan, EM Airoldi, A Clauset. Stacking Models for Nearly Optimal Link Prediction in Complex Networks. Proceedings of the National Academy of Sciences. In press, 2020.</p>
  </li>

<li>
  <p>    A Franks✝, EM Airoldi, DB Rubin. Non-standard conditionally specified models for non-ignorable missing data mechanisms. Proceedings of the National Academy of Sciences. 117, 19045-19053, 2020.</p>
  </li>

<li>
  <p>    J Pouget-Abadie✝, M Saveski, G Saint-Jacques✝, W Duan, Y Xu, S Ghosh, EM Airoldi. Testing for arbitrary interference on experimentation platforms. Biometrika. 106, 929–940, 2019.</p>
  </li>

<li>
  <p>     G Iosifidis, Y Charette, EM Airoldi, G Littera, L Tassiulas, NA Christakis. Short network cycle motifs and economic efficiency in a novel monetary system. Nature Human Behavior. 2, 822-829, 2018.</p>
  </li>

<li>
  <p>    P Bhattacharya, TQ Phan, X Bai, EM Airoldi. A co-evolution model of network structure and user behavior: The case of content generation in online social networks. Information Systems Research. 30, 117-132, 2018.</p>
  </li>

<li>
  <p>     GW Basse✝, EM Airoldi. Model-assisted design of experiments for network-correlated outcomes suggests new notions of network balance. Biometrika. 105, 849-858, 2018.</p>
  </li>

<li>
  <p>    A Isakov, JH Fowler, EM Airoldi, NA Christakis. The structure of negative social ties in rural village networks. Sociological Science. 6, 197-218, 2019.</p>
  </li>

<li>
  <p>    EK Kao✝, ST Smith, EM Airoldi. Hybrid Mixed-Membership Blockmodels for inference on realistic network interactions. IEEE Transactions on Network Science and Engineering. 6, 336-350, 2019.</p>
  </li>

<li>
  <p>    J Pouget-Abadie✝, DK Parkes, V Mirrokni, EM Airoldi. Optimizing cluster-based randomized experiments under a monotonicity assumption. ACM International Conference on Knowledge Discovery & Data Mining (KDD), London, UK, 2018.</p>
  </li>

<li>
  <p>    GW Basse✝, EM Airoldi. Limitations of design-based causal inference and A/B testing under arbitrary and network interference. Sociological Methodology. 48, 136-151, 2018.</p>
  </li>

<li>
  <p>     D Malioutov, T Chen, J Jaffe, EM Airoldi, S Carr, B Budnik, N Slavov✝. Quantifying homologous proteins and proteoforms. Molecular and Cellular Proteomics. 2018, no. TIR118.0009.</p>
  </li>

<li>
  <p>     AM Franks✝, F Markowetz, EM Airoldi. Refining cellular pathway models using an ensemble of heterogeneous data sources. Annals of Applied Statistics. 12, 1361-1384, 2018.</p>
  </li>

<li>
  <p>     P Toulis✝, EM Airoldi. Asymptotic and finite-sample properties of estimators based on stochastic gradients. Annals of Statistics. 45, 1694–1727, 2017.</p>
  </li>

<li>
  <p>     D Jie, Q Han, EM Airoldi, V Tarokh. Sequential adaptive nonlinear modeling of time series. IEEE Transactions on Signal Processing. 65, 4994-5005, 2017.</p>
  </li>

<li>
  <p>     AM Franks✝, EM Airoldi, N Slavov✝. Post-transcriptional regulation across human tissues. PLoS Computational Biology. 13, e1005535, 2017.</p>
  </li>

<li>
  <p>     HB Shakya, D Stafford, DA Hughes, T Keegan, R Negron, J Broome, M McKnight, L Nicoll, J Nelson, E Iriarte, M Ordonez, EM Airoldi, JH Fowler, NA Christakis. Exploiting social influence to magnify population-level behavior change in maternal and child health: Study protocol for a randomized controlled trial of network targeting algorithms in rural Honduras. BMJ Open. 7, e012996, 2016.</p>
  </li>

<li>
  <p>     P Toulis✝, D Tran✝, EM Airoldi. Towards stability and optimality in stochastic gradient descent. Journal of Machine Learning Research, W&CP (AI&Stats). 2016</p>
  </li>

<li>
  <p>    H Cavusoglu, TQ Phan✝, H Cavusoglu, EM Airoldi. Assessing the effects of granular privacy controls on content sharing and disclosure on Facebook. Information Systems Research. 27, 848-879, 2016.</p>
  </li>

<li>
  <p>    A Volfovsky✝, EM Airoldi. Sharp total variation bounds for finitely exchangeable arrays. Statistics & Probability Letters. 114, 54-59, 2016.</p>
  </li>

<li>
  <p>    S Lunagomez✝, S Mukherjee, R Wolpert, EM Airoldi. Geometric representations of distributions on hypergraphs. Journal of the American Statistical Association. 112, 363-383, 2016.</p>
  </li>

<li>
  <p>    M Roberts✝, B Stewart✝, EM Airoldi. A topic model for experimentation in the social sciences. Journal of the American Statistical Association. 111, 988-1003, 2016.</p>
  </li>

<li>
  <p>    EM Airoldi, J Bischof✝. Improving and evaluating topic models and other models of text (with discussion). Journal of the American Statistical Association. 111, 1381-1403, 2016.</p>
  </li>

<li>
  <p>   X Zhou, AW Blocker✝, EM Airoldi, EK O’Shea. A computational approach to map nucleosome positions and alternative chromatin states with base pair resolution. eLife, 5, e16970, 2016.</p>
  </li>

<li>
  <p>    E Solis✝, J Pandey, X Zheng, D Jin, P Gupta, EM Airoldi, D Pincus, V Denic. Defining the essential function of yeast Hsf1 reveals a compact transcriptional program for maintaining eukaryotic proteostasis. Molecular Cell. 63, 60-71, 2016.</p>
  </li>

<li>
  <p>     EM Airoldi, TB Hashimoto✝, N Brandt, T Bahmani, N Athanasiadou, DJ Gresham. Steady-state and dynamic gene expression programs in Saccharomyces cerevisiae in response to variation in environmental nitrogen. Molecular Biology of the Cell. 27, 1383-1396, 2016.</p>
  </li>

<li>
  <p>    DJ Klionsky, et al. Guidelines for the use and interpretation of assays for monitoring autophagy (3rd edition). Authophagy, 12, 1–222, 2016.</p>
  </li>

<li>
  <p>    AW Blocker✝, EM Airoldi. Template-based models for genome-wide analysis of next-generation sequencing data at base-pair resolution. Journal of the American Statistical Association. 111,  967-987, 2016. (Microsoft best student paper award at the New England Statistics Symposium, 2012.)</p>
  </li>

<li>
  <p>    N Slavov✝, S Semrau, EM Airoldi, B Budnik, A Van Oudenaarden. Differential stoichiometry among core ribosomal proteins. Cell Reports, 13, 865–873, 2016.</p>
  </li>

<li>
  <p>     P Toulis✝, EM Airoldi. Scalable estimation strategies based on stochastic approximations: Classical results and new insights. Statistics and Computing. 25, 781–795, 2015.</p>
  </li>

<li>
  <p>     D Tran✝, DM Blei, EM Airoldi. Copula variational inference. Neural and Information Processing Systems (NIPS). 2015.</p>
  </li>

<li>
  <p>     CQ Han✝, K Xu, EM Airoldi. Consistent estimation of dynamic and multi-layer block models. Journal of Machine Learning Research, W&CP (ICML). 37, 1511–1520, 2015.</p>
  </li>

<li>
  <p>     R Silva, S Kang, EM Airoldi. Predicting traffic volumes and quantifying the effects of shocks in massive transportation systems. Proceedings of the National Academy of Sciences. 112, 5643–5648, 2015.</p>
  </li>

<li>
  <p>    TQ Phan✝, EM Airoldi. A natural experiment of social network formation and dynamics. Proceedings of the National Academy of Sciences. 112, 6595–6600, 2015.</p>
  </li>

<li>
  <p>  EJ Wallace✝, JL Kear-Scott, EV Pilipenko, MH Schwartz, PR Laskowski, AE Rojek, CD Katanski, JA Riback, MF Dion, AM Franks✝, EM Airoldi, T Pan, BA Budnik, DA Drummond. Heat stress triggers formation of reversible, specific, functional aggregates of endogenous proteins in yeast. Cell, 162, 1286–1298, 2015.</p>
  </li>

<li>
  <p>    J Rameseder, K Krismer, Y Dayma, T Ehrenberger, MK Hwang, EM Airoldi, SR Floyd, MB Yaffe. A multivariate computational method to analyze high-content RNAi screening data. Journal of Biomolecular Screening. 20, 985-997, 2015.</p>
  </li>

<li>
  <p>    AM Franks✝, G Csardi✝, DS Choi✝, EM Airoldi, DA Drummond. Accounting for experimental noise reveals that mRNA levels, amplified by post-transcriptional processes, largely determine steady-state protein levels in yeast. PLoS Genetics. 11, e1005206, 2015.</p>
  </li>

<li>
  <p>     Y Katz✝, ET Wang, J Stilterra, S Schwartz, B Wong, H Thorvaldsdóttir, JT Robinson, JP Mesirov, EM Airoldi, CB Burge. Quantitative visualization of alternative isoform expression from RNA-seq data. Bioinformatics. 31, 2400–2402, 2015.</p>
  </li>

<li>
  <p>     AM Franks✝, G Csardi✝, DA Drummond, EM Airoldi. Estimating a structured covariance matrix from multi-lab measurements in high-throughput biology. Journal of the American Statistical Association. 110, 27-44, 2015. (IBM Best student paper award at the New England Statistics Symposium, 2013.)</p>
  </li>

<li>
  <p>    P Toulis✝, J Rennie, EM Airoldi. Statistical analysis of stochastic gradient methods for generalized linear models. Journal of Machine Learning Research, W&CP (ICML). 32, 667-675, 2014.</p>
  </li>

<li>
  <p>    E Azizi✝, EM Airoldi, JE Galagan. Learning modular structures from network data and node variables. Journal of Machine Learning Research, W&CP (ICML). 32, 1440-1448, 2014. (Best student paper award at the New England Statistics Symposium 2014.)</p>
  </li>

<li>
  <p>     S Chan✝, EM Airoldi. A consistent histogram estimator for exchangeable graph models. Journal of Machine Learning Research, W&CP (ICML). 32, 208-216, 2014.</p>
  </li>

<li>
  <p>     JJ Yang✝, CQ Han✝, EM Airoldi. Non-parametric estimation and testing of exchangeable graph models. Journal of Machine Learning Research, W&CP (AI&Stats). 30, 1060-1067, 2014.</p>
  </li>

<li>
  <p>    EM Airoldi, T Costa✝, F Bassetti, F Leisen, M Guindani. Generalized species sampling priors with latent beta reinforcements. Journal of the American Statistical Association, 109, 1466-1480, 2014.</p>
  </li>

<li>
  <p>    Y Katz✝, F Li, N Lambert, EM Sokol, W-L Tam, AW Cheng, EM Airoldi, CJ Lengner, PB Gupta, Z Yu, R Jaenisch, CB Burge. Musashi proteins are post-transcriptional regulators of the epithelial-luminal cell state. eLife, 3, e03915, 2014.</p>
  </li>

<li>
  <p>    N Slavov✝, B Budnik, D Schwab, EM Airoldi, A van Oudenaarden. Constant growth rate can be supported by decreasing energy flux and increasing aerobic glycolysis. Cell Reports. 7, 705-714, 2014.</p>
  </li>

<li>
  <p>    EM Airoldi, T Costa✝, S Chan✝. Stochastic blockmodel approximation of a graphon: Theory and consistent estimation. Neural and Information Processing Systems (NIPS). 2013.</p>
  </li>

<li>
  <p>    S Chan✝, T Costa✝, EM Airoldi. Estimation of exchangeable graph models by stochastic blockmodel approximation. IEEE Global Conference on Signal and Information Processing. 2013.</p>
  </li>

<li>
  <p>     EM Airoldi, AW Blocker✝. Estimating latent processes on a network from indirect measurements. Journal of the American Statistical Association.108, 149-164, 2013.</p>
  </li>

<li>
  <p>     EM Airoldi, X Wang, X Lin. Multiway blockmodels for analyzing coordinated high dimensional responses. Annals of Applied Statistics. 7, 2431-2457, 2013.</p>
  </li>

<li>
  <p>     KA Geiler-Samerotte, T Hashimoto, MF Dion, BA Budnik, EM Airoldi, DA Drummond. Quantifying condition-dependent intracellular protein levels enables high-precision fitness estimates. PLoS One. 8, e75320, 2013.</p>
  </li>

<li>
  <p>     EJ Wallace✝, EM Airoldi, DA Drummond. Estimating selection on synonymous codon usage. Molecular Biology and Evolution. 30, 1438-1453, 2013.</p>
  </li>

<li>
  <p>     H Azari✝, EM Airoldi. Graphlet decomposition of a weighted network. Journal of Machine Learning Research, W&CP (AI&Stats). 22, 54-63, 2012. (Microsoft Best student paper award at the New England Statistics Symposium 2012.)</p>
  </li>

<li>
  <p>     DS Choi✝, PJ Wolfe, EM Airoldi. Stochastic blockmodels with growing number of classes. Biometrika. 99, 273-284, 2012. </p>
  </li>

<li>
  <p>     N Slavov✝, EM Airoldi, A van Oudenaarden, D Botstein. A conserved cell growth cycle can account for the environmental stress responses of divergent eukaryotes. Molecular Biology of the Cell. 23, 1986-1997, 2012.</p>
  </li>

<li>
  <p>    EM Airoldi, DS Choi✝, PJ Wolfe. Confidence sets for network structure. Statistical Analysis and Data Mining. 4, 461-469, 2011. </p>
  </li>

<li>
  <p>     AD Shieh✝, TB Hashimoto✝, EM Airoldi. Tree preserving embedding. Proceedings of the National Academy of Sciences. 108, 16916-16921, 2011. </p>
  </li>

<li>
  <p>     EM Airoldi, X Bai, KM Carley. Network sampling and classification: An investigation of network model representations. Decision Support Systems. 51, 506-518, 2011. </p>
  </li>

<li>
  <p>     DS Choi✝, EM Airoldi, PJ Wolfe. Confidence sets for network structure. Neural and Information Processing Systems (NIPS). 24, 2097-2105, 2011. </p>
  </li>

<li>
  <p>    AW Blocker✝, EM Airoldi. Deconvolution of mixing time series on a graph. Uncertainty in Artificial Intelligence (UAI). 27, 51-60, 2011. (IBM Best student paper award at the New England Statistics Symposium 2011.)</p>
  </li>

<li>
  <p>     AD Shieh✝, TB Hashimoto✝, EM Airoldi. Tree preserving embedding. International Conference on Machine Learning (ICML). 28, 753-760, 2011.</p>
  </li> 

<li>
  <p>    EM Airoldi, B Haas✝. Polytope samplers for ill-posed inverse problems. Journal of Machine Learning Research, W&CP (AI&Stats). 15, 110-118, 2011. </p>
  </li>

<li>
  <p>     EM Airoldi, KA Heller, R Silva. Bayesian ranking by analogy (BRA): Small sets of interacting proteins suggest functional linkage mechanisms. Bioinformatics. 27, i374-i382, 2011. </p>
  </li>

<li>
  <p>    R Silva, KA Heller, Z Ghahramani, EM Airoldi. Ranking relations using analogies in biological and information networks. Annals of Applied Statistics. 4, 615-644, 2010. </p>
  </li>

<li>
  <p>     Y Katz✝, E Wang, EM Airoldi, CB Burge. Analysis and design of RNA sequencing experiments for identifying mRNA isoform regulation. Nature Methods. 7, 1009-1015, 2010. </p>
  </li>

<li>
  <p>     F Markowetz, KW Mulder, EM Airoldi, IR Lemischka, OG Troyanskaya. Mapping dynamic histone acetylation patterns to gene expression in Nanog-depleted murine embryonic stem cells. PLoS Computational Biology. 6, e1001034, 2010. </p>
  </li>

<li>
  <p>     EM Airoldi, EA Erosheva, SE Fienberg, C Joutard, TM Love, S Shringarpure. Reconceptualizing the classification of PNAS articles. Proceedings of the National Academy of Sciences. 107, 20899-20904, 2010. (Featured article.)</p>
  </li>

<li>
  <p>     A Goldenberg, AX Zheng, SE Fienberg, EM Airoldi. A survey of statistical network models. Foundations and Trends in Machine Learning. 2, 129-233, 2009. </p>
  </li>

<li>
  <p>    R Lu, F Markowetz, RD Unwin, JT Leek, EM Airoldi, BD MacArthur, A Lachmann, R Rozov, A Ma’ayan, LA Boyer, OG Troyanskaya, AD Whetton, IR Lemischka. Systems-level dynamic analyses of fate change in murine embryonic stem cells. Nature. 462, 358-362, 2009. </p>
  </li>

<li>
  <p>    Z Barutcuoglu, EM Airoldi, V Dumeaux, RE Schapire, OG Troyanskaya. Aneuploidy prediction and tumor classification with heterogeneous hidden conditional random fields. Bioinformatics. 25, 1307-1313, 2009.</p>
  </li> 

<li>
  <p>     EM Airoldi, C Huttenhower, DJ Gresham, C Lu, A Caudy, M Dunham, JR Broach, D Botstein, OG Troyanskaya. Predicting cellular growth with gene expression signatures. PLoS Computational Biology. 5, e100025, 2009. </p>
  </li>

<li>
  <p>     EM Airoldi, DM Blei, SE Fienberg, EP Xing. Mixed-membership stochastic blockmodels. Journal of Machine Learning Research. 9, 1981-2014, 2008. </p>
  </li>

<li>
  <p>     EM Airoldi, DM Blei, SE Fienberg, EP Xing. Mixed-membership stochastic blockmodels. Neural and Information Processing Systems (NIPS). 2008. </p>
  </li>

<li>
  <p>    MJ Brauer, C Huttenhower, EM Airoldi, R Rosenstein, JC Matese, D Gresham, VM Boer, OG Troyanskaya, D Botstein. Coordination of growth rate, cell cycle, stress response and metabolic activity in yeast. Molecular Biology of the Cell. 19, 352-367, 2008. </p>
  </li>

<li>
  <p>     EM Airoldi. Getting started in probabilistic graphical models. PLoS Computational Biology, 3, e252, 2007. </p>
  </li>

<li>
  <p>     EM Airoldi, SE Fienberg, KK Skinner. Whose ideas? Whose words? Authorship of the Ronald Reagan radio addresses. Political Science & Politics, 40, 501-506, 2007. (NYT op-ed by Skinner & Rice.)</p>
  </li>

<li>
  <p>     EM Airoldi, DM Blei, SE Fienberg, EP Xing. Combining stochastic blockmodels and mixed membership for statistical network analysis. Statistical Network Analysis (Revised Selected Papers), Lecture Notes in Computer Science, vol. 4503, 57-74, 2007. </p>
  </li>

<li>
  <p>     EM Airoldi, DM Blei, SE Fienberg, EP Xing. Mixed membership stochastic blockmodels for relational data with application to protein-protein interactions. International Biometrics Society Annual Meeting, East North American Region (ENAR), Atlanta GA, 2006. </p>
  </li>

<li>
  <p>     EM Airoldi, A Anderson, SE Fienberg, KK Skinner. Who wrote Ronald Reagan's radio addresses? Bayesian Analysis, 1, 289-320, 2006. </p>
  </li>

<li>
  <p>     EM Airoldi, KM Carley. Sampling algorithms for pure network topologies: Stability and separability of metric embeddings. ACM SIGKDD Explorations. 7, 13-22, 2005. </p>
  </li>

<li>
  <p>     B Malin, EM Airoldi, KM Carley. A social network analysis model for name disambiguation in lists. Journal of Computational and Mathematical Organization Theory. 11, 119-139, 2005. </p>
  </li>

<li>
  <p>     EM Airoldi, C Faloutsos. Network tomography: Recovering latent time series from their observed sums. ACM International Conference on Knowledge Discovery & Data Mining (KDD), Seattle WA, 2004. </p>
  </li>

</ol>


**Edited Books and Monographs**

2.    EM Airoldi, DM Blei, EA Erosheva, SE Fienberg, editors. Handbook of Mixed Membership Mod-els and Their Applications. CRC Press, 2014. 

1.    EM Airoldi, DM Blei, SE Fienberg, EP Xing, editors. Statistical Network Analysis: Models, Is-sues, and New Directions. Lecture Notes in Computer Science, volume 4503. Springer, 2007. 


**Other Scholarly Writings** 

*Discussions*

2.    A Feller✝, EM Airoldi. Discussion of Hennig and Liao “How to find an appropriate clustering for mixed type variables with application to socio-economic stratification”. Journal of the Royal Statistical Society, Series C. 62, 347-349, 2013.

1.    EM Airoldi. Discussion of Handcock, Raftery and Tantrum “Model-based clustering for social networks”. Journal of the Royal Statistical Society, Series A, 170, Part 2, 30-31, 2007. 

*Technical reports*

3.    T Hashimoto✝, DJ Gresham, D Botstein, EM Airoldi. A linear model framework for genome-scale functional analysis based on the gene ontology. Princeton University, 2009.

2.    EM Airoldi. Who Wrote Ronald Reagan's Radio Addresses? Carnegie Mellon University, 2004. 

1.    EM Airoldi. Advances in Network Tomography. Carnegie Mellon University, 2003.

*Dissertations*

2.    EM Airoldi. Bayesian mixed-membership models of complex and evolving networks. School of Computer Science, Carnegie Mellon University, 2007. 

1.    EM Airoldi. Weak convergence of probability measures, with an application to Girone-Cifarelli (in Italian). Institute for Quantitative Methods, Bocconi University, 1998. 


**Open-Source Software** 

8.    SGD, an R package. Implements stochastic approximation methods for parameter estimation. 

7.    iGraph 1.0.0, an R package, and Python and C libraries. Implements methods to analyze graphs.

6.    ABQuant, R functions, and Python and C libraries. Implements methods for estimating absolute protein concentrations from Mass Spectrometry data. (Single processor and MPI.)

5.    CPlate, R functions, and Python and C libraries. Implements methods for estimating nucleosomes insertions, deletions, and their primary and alternative positions within a population of cells from paired-end mRNA-sequencing data. (Single processor and MPI.)

4.    MISO, Python and C libraries. Implements methods for estimating expression of alternatively spliced isoforms from paired-end mRNA-sequencing data. (Single processor and MPI.)

3.    TPE, an R package. Implements tree preserving embedding.

2.    networkTomography, an R package. Implements methods for estimating latent time series on a graph, including those published in JASA. Includes 3 data sets of point-to-point time series. 

1.    Tetrahedron 3D, a JAVA applet to explore the geometry of 2x2 contingency tables. 

